tests/yast2_cmd/yast_rdp.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: this test checks that YaST2's RDP module is behaving
+#          correctly in cmdline mode  and verifying that RDP service
+#          has been successfully set.
+# Maintainer: Jun Wang <jgwang@suse.com>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+use version_utils 'is_sle';
+
+sub run {
+    select_console 'root-console';
+
+    # yast2-rdp was added after SLE15.
+    if (is_sle('15+')) {
+        zypper_call("in yast2-rdp", exitcode => [0, 102, 103]);
+    } else {
+        die "This version of product is NOT supported.";
```

Comment:
If test should run only on 15+ then define it in main_common.pm, no need to fail on sle12

Reply:
I see, your  point is good. And I have improve the code.